pytest_xvfb.py

```diff
@@ -130,8 +131,12 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
-    if config.getoption('--no-xvfb') or not xvfb_available():
+    if config.getoption('--no-xvfb'):
         config.xvfb = None
+    elif not xvfb_available():
+        config.xvfb = None
+        warnings.warn(
+            'Pytest-Xvfb could not find Xvfb installation. Please see that it is installed.')
```

Comment:
As mentioned in https://github.com/The-Compiler/pytest-xvfb/issues/2#issuecomment-185709984 this should also not be shown in some circumstances:

- When not on Linux (as Xvfb is not available there anyways)
- When `$DISPLAY` is unset (running on some system without GUI, tests will fail or be skipped anyways)
- When `--no-xvfb` is given